query/src/frontend/reorg.rs

```diff
@@ -73,10 +73,14 @@ impl ReorgPlanner {
         let ScanPlan {
             plan_builder,
             provider,
-        } = self.scan_and_sort_plan(chunks, output_sort)?;
+        } = self.scan_and_sort_plan(chunks, output_sort.clone())?;
+
+        let mut schema = provider.iox_schema();
 
-        // TODO: Set sort key on schema
-        let schema = provider.iox_schema();
+        trace!("Schema before setting sort key: {:#?}", schema);
```

Comment:
When logging it is preferable to keep the log message fixed and then provide structured keys and values. This will help when processing logs or grepping for specific log events.

```suggestion
        trace!(input_schema=?schema, "Setting sort key on schema");
```

Reply:
@e-dard Is this equivalent to your suggestion?
trace!("Setting sort key on schema - input_schema:{:#?}", schema);
I want to have the structure that is easy to read and the `?schema` does not provide that

Reply:
It's not quite the same. The idea of using structured fields in your logs is that they comply to the logfmt spec, which means any parser can split out the fields into separate values. If we were feeding these log lines into IOx for example we would be able to put them in different columns. 

The message itself is one field and the input schema would be another.

Reply:
Let me keep it your format, then. Thanks @e-dard